ui/apps/platform/src/hooks/useAnalytics.ts

```diff
@@ -155,6 +159,21 @@ export type AnalyticsEvent =
               CVE_STATUS_NOT_FIXABLE: AnalyticsBoolean;
           };
       }
+    | {
+          event: typeof WORKLOAD_CVE_DEFERRAL_EXCEPTION_REQUESTED;
+          properties:
+              | {
+                    expiryType: 'TIME';
+                    expiresOn: string | null;
```

Comment:
Is `expiresOn` an absolute date?

What is return on investment for future analysis:
* report `days` for built in options like 14, 30, 60, 90
* report for custom `date` the relative number of `days` to the date? or if both, `date` as just yyyy-mm-dd
    Imagining 2 cases
        * evidence if a missing relative days option
        * evidence of organization patters like beginning of month or end of month

https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/components/ExceptionRequestModal/ExpiryField.tsx#L88-L106
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/components/ExceptionRequestModal/ExpiryField.tsx#L116-L145

Reply:
Oh yea, that's an interesting point. I know the backend expects an `expiresOn`, but in the UI we offer more human-friendly values to show when to expire (ie. 14, 30, 60, 90 days). I think Mark has a point that we should probably track what options are being selected because the `expiresOn` string won't give us too much information as to the user's habits.

Reply:
Yes! Good call, I was thinking `expiresOn` was the configurable day value, but it is a timestamp. Agree that the timestamp is of little to no use, let me take a look at this again and see if we can easily capture the `30d`, `60d`, etc.

Reply:
Ok I'm going to update the testing section shortly. This accounts for relative `expiryDays` instead of the exact date of expiration, which is useful to the API but not so much to telemetry data.

For the `CUSTOM_DATE` expiry type, I'm doing the conversion myself on the UI. Even when the user selects this option, I don't believe the date in-particular is useful, but the distance in the future from the current date may be.